Amplify/Categories/Predictions/Request/Convert/Convert+SpeechToText.swift

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+extension Predictions.Convert {
+    public enum SpeechToText {}
+}
+
+extension Predictions.Convert.Request where
+Input == URL,
```

Comment:
nit: the indentation in these types is kinda messed up ... maybe?  🤔

Reply:
That's actually how Xcode handles newline breaks between where clauses by default. 🙃 
I could indent them, but a future Re-Indent (`^ I`) would push them back.